composer.json

```diff
@@ -13,7 +13,7 @@
 		],
 		"test": [
 			"parallel-lint . --exclude vendor --exclude node_modules",
-			"phpcs -p -s",
+			"phpcs -p -s  --ignore='tests/api-testing'",
```

Comment:
Hm, is this necessary? I assume we shouldn’t have any PHP files in that directory anyways.

Reply:
If you look [lint run from previous commit](https://github.com/wmde/WikibaseReconcileEdit/runs/2622625960), It's annoyingly picking up the requestHelper.js

Reply:
WTF. ok